src/main/java/com/epam/lab/payments/model/Card.java

```diff
@@ -16,9 +16,10 @@
 public class Card {
     @Id
     @GeneratedValue
-    private Long id;
+    public Long id;
 
-    private String cardholderName;
-    private LocalDate expiration;
-    private Integer securityCode;
+    public String cardHolderName;
+    public LocalDate expiration;
+    public Integer securityCode;
+    public Long accountId;
```

Comment:
Why reference type?

Reply:
JPA can work with LocalDate, isn't it? And there is reference type in all guide.